teps/0044-decouple-task-composition-from-scheduling.md

```diff
@@ -504,156 +705,74 @@ Cons:
 * Might be hard to reason about what is executed together
 * Might be hard to reason about what which Tasks can be combined in a group and which can't
```

Comment:
I am not sure I understand this one now. In the end, tasks are "owned" by the user, they are the one with that knowledge.